run.py

```diff
@@ -7,16 +7,18 @@
     z5gExperiment,
 )
 from experiments.toptagging.experiment import QGTaggingExperiment
+from experiments.toptagging.jetclassexperiment import JetClassTaggingExperiment
 
-
-@hydra.main(config_path="config", config_name="amplitudes", version_base=None)
+@hydra.main(config_path="config", config_name="jctagging", version_base=None)
```

Comment:
Before merging into main, I would undo this change again to have 'amplitudes' as the default. Happy to keep it as it is until then. Note that you can specify the config_name with --config_name=XYZ or -cn XYZ